test/e2e/auth/service_accounts.go

```diff
@@ -673,7 +673,14 @@ var _ = SIGDescribe("ServiceAccounts", func() {
 		}
 	})
 
-	ginkgo.It("should support OIDC discovery of service account issuer [Feature:ServiceAccountIssuerDiscovery]", func() {
+	/*
+		   Release: v1.21
+		   Testname: OIDC Discovery (ServiceAccountIssuerDiscovery)
+		   Description: Ensure kube-apiserver serves correct OIDC discovery
+						endpoints by deploying a Pod that verifies its own
+						token against these endpoints.
+	*/
+	framework.ConformanceIt("should support OIDC discovery of service account issuer [Feature:ServiceAccountIssuerDiscovery]", func() {
```

Comment:
drop `[Feature:ServiceAccountIssuerDiscovery]`. feature tagged tests are excluded from the main e2e by default because they are assumed to need special config

Reply:
Done